restore/README.md

```diff
@@ -9,6 +9,8 @@ The restore action restores a cache. It works similarly to the `cache` action ex
 * `key` - An explicit key for a cache entry. See [creating a cache key](../README.md#creating-a-cache-key).
 * `path` - A list of files, directories, and wildcard patterns to restore. See [`@actions/glob`](https://github.com/actions/toolkit/tree/main/packages/glob) for supported patterns.
 * `restore-keys` - An ordered list of prefix-matched keys to use for restoring stale cache if no cache hit occurred for key.
+* `enableCrossOsArchive` - An optional boolean when enabled, allows windows runners to restore caches that were saved on other platforms. Default: `false`
+* `enableCrossArchArchive` - An optional boolean when enabled, allows runners of different architectures to restore caches that were saved on other architectures. Default: `false`
```

Comment:
Please mention this is an extra flag. GitHub action doesn't support it and what's the behaviour like there.

Reply:
updated